docker/client.py

```diff
@@ -240,10 +242,10 @@ def _multiplexed_response_stream_helper(self, response):
         # Disable timeout on the underlying socket to prevent
         # Read timed out(s) for long running processes
```

Comment:
Looks like this appears twice now. We probably only want to keep the second occurrence.